app/models/miq_expression.rb

```diff
@@ -1026,22 +1026,22 @@ def self.evaluate(expression, obj, _inputs = {}, tz = nil)
     result
   end
 
-  def evaluate(obj, inputs = {}, tz = nil)
-    self.class.evaluate(self, obj, inputs, tz)
+  def evaluate(obj, tz = nil)
+    self.class.evaluate(self, obj, tz)
```

Comment:
Can an instance call a private class method? I don't think it can:
```
irb(main):001:0> class Foo
irb(main):002:1> def self.bar
irb(main):003:2> puts "boop!"
irb(main):004:2> end
irb(main):005:1> private_class_method :bar
irb(main):006:1> def bar
irb(main):007:2> self.class.bar
irb(main):008:2> end
irb(main):009:1> end
=> :bar
irb(main):010:0> Foo.bar
NoMethodError: private method `bar' called for Foo:Class
	from (irb):10
	from /home/tim/.rubies/ruby-2.2.4/bin/irb:11:in `<main>'
irb(main):011:0> Foo.new.bar
NoMethodError: private method `bar' called for Foo:Class
	from (irb):7:in `bar'
	from (irb):11
	from /home/tim/.rubies/ruby-2.2.4/bin/irb:11:in `<main>'
```